modules/integration/tests-common/admin-clients/src/main/java/org/wso2/mb/integration/common/clients/operations/utils/AndesClientUtils.java

```diff
@@ -348,6 +342,22 @@ public static void createMockFile(String filePathToRead, String filePathToCreate
         } catch (IOException e) {
             log.error("Error. File to print received messages is not provided", e);
         }
```

Comment:
No need to say specifically error. "Error" part would come from logging. Shall we remove error?